### DISCUSSION TITLE
notes/sc-i8042prt/

### DISCUSSION THREAD

**Discussion by giscus** [x2rs/x2rs.github.io#23] [Giscus Comments]
# notes/sc-i8042prt/

My personal documentation website.

https://x2rs.github.io/notes/sc-i8042prt/

**Comment by x2rs** [comment]
在 markdown 代码块中写 cmd，被 mkdocs-material 识别成 text only，那应该写什么呢？

**Comment by co-knight** [comment]
闲逛看到了之前没搞成的禁用自带键盘，很有趣呢，虽然也不是很有用处hhhhh

如果要显示 shell command 可以试试 console
```console
C:\user\coknight>echo hello
```
当然一般是linux上的
```console
coknight@bar:~$ whoami
```
